src/pynxtools/dataconverter/validation.py

```diff
@@ -649,15 +1429,15 @@ def handle_field(node: NexusNode, keys: Mapping[str, Any], prev_path: str):
 
             # Check general validity
             mapping[variant_path] = is_valid_data_field(
-                mapping[variant_path],
+                keys[variant],
```

Comment:
Why this change?

Reply:
It just appeared somewhat inconsistent with the use if mapping[variant_path] further below. But if it contains the same, fine for me.